### PR TITLE
DOC: Fix dark mode styles

### LINKED ISSUE #48527: DOC: Fix styles for dark mode in docs home
xref: #42744

We'll be making the pandas documentation have a dark mode soon. But some boxes in the docs home seem to have a white background, instead of a transparent one:

Screenshot_20220913_143553

We should remove any custom style that is preventing the dark mode to work properly.

The changes can be tested against this branch to see if they work properly: #48285

### PR DESCRIPTION
**Pull request by anushkabishnoi**
closes #48527

### CODE REVIEW

**Comment by mroeschke** [comment]
Is this in reference to an existing Github issue?

File: doc/source/_static/css/getting_started.css [hunk 1]
```diff
@@ -236,7 +236,7 @@ ul.task-bullet > li > p:first-child {
 
 .tutorial-card .card-header {
   cursor: pointer;
-  background-color: white;
+  background-color: transparent;
```
Comment (tuhinmallick) [inline doc/source/_static/css/getting_started.css:239]: Is this related to #48527 ?

Comment (anushkabishnoi) [reply doc/source/_static/css/getting_started.css:239]: yes it is, sorry for not mentioning this (first timer here)

**Comment by anushkabishnoi** [comment]
> Is this in reference to an existing Github issue?

yes, it's in reference to https://github.com/pandas-dev/pandas/issues/42744 
just realising I should've mentioned that, sorry for the misunderstanding

**Comment by tuhinmallick** [comment]
It is already closed

**Comment by anushkabishnoi** [comment]
> It is already closed

But there was a portion of the site still not corrected, ans that's what I modified. The issue is still open because of that.

**Comment by datapythonista** [comment]
Preview docs at: https://pandas.pydata.org/preview/50936/

**Comment by datapythonista** [comment]
Thanks @anushkabishnoi for working on this.

Looks like the collapsed blocks are fixed by this, but when you expand it still has white background: https://pandas.pydata.org/preview/50936/docs/getting_started/index.html

Do you want to have a look and see if you can also fix that?

**Comment by datapythonista** [comment]
@anushkabishnoi do you have time to address the comments above?

**Comment by anushkabishnoi** [comment]
hey I'm sorry for not responding earlier, I tried to work out the other thing you pointed out, and I can't seem to figure it out. 
Is it fine if we just merge the original pr, or do i try the other thing again?

Thanks in advance.

**Comment by datapythonista** [comment]
What's the exact problem? Seems like it should just be repeating what you did to the style of the other component.

**Review by PKNaveen — CHANGES_REQUESTED** [review]
To remove the white background while expanding replace line 242 with this
```
.tutorial-card .card-body {
  background-color: #F0F0F0;
  background-color: transparent;
}
```

**Comment by anushkabishnoi** [comment]
I've pushed a commit, the only issue still prevailing - the text on the svg needs to change with dark and light modes. I don't have eperience working with SVGs, so if you guys can help me out, that would be great.

(this is what I mentioned not being able to figure out)

File: doc/source/_static/css/getting_started.css [hunk 2]
```diff
@@ -236,11 +236,12 @@ ul.task-bullet > li > p:first-child {
...
 }
 
 .tutorial-card .card-body {
   background-color: #F0F0F0;
```
Comment (datapythonista) [inline doc/source/_static/css/getting_started.css:243]: Can you remove this line please?

Comment (anushkabishnoi) [reply doc/source/_static/css/getting_started.css:243]: sure

Comment (anushkabishnoi) [reply doc/source/_static/css/getting_started.css:243]: the change isn't being acknowledged as a modification, so it's not commiting (working tree clean)

Comment (datapythonista) [inline doc/source/_static/css/getting_started.css:243]:
```suggestion
```

**Comment by datapythonista** [comment]
Deleted the line myself. So far doesn't seem to be working, let's see if with this new change: https://pandas.pydata.org/preview/50936/docs/getting_started/index.html

Would be good to test things locally, since the CI is not very fast.

**Review by datapythonista — APPROVED** [review]
Looks like removing the original line worked, this is working now. https://pandas.pydata.org/preview/50936/docs/getting_started/index.html

**Review by mroeschke — APPROVED** [review]
